docker/orchestrator/build.sh

```diff
@@ -23,7 +23,7 @@ script="go get vitess.io/vitess/go/cmd/vtctlclient && \
   go install github.com/openark/orchestrator/go/cmd/orchestrator"
 
 echo "Building orchestrator..."
-docker run -ti --name=vt_orc_build golang:1.14.4-stretch bash -c "$script"
+docker run -ti --name=vt_orc_build golang:1.14.4-buster bash -c "$script"
```

Comment:
This is technically not needed as orchestrator has its own build process but I think it's 👍 if we align the versions across the whole repo.